src/main/scala/de/frosner/dds/core/DDS.scala

```diff
@@ -311,12 +311,18 @@ object DDS {
     category = "Spark Core",
     shortDescription = "Plots a histogram of a numerical RDD for the given number of buckets",
     longDescription = "Plots a histogram of a numerical RDD for the given number of buckets. " +
-      "The number of buckets parameter is optional having the default value of 10.",
-    parameters = "values: RDD[NumericValue], (optional) numBuckets: Int"
+      "The number of buckets parameter is optional - if omitted, Sturge's formula is used to determine an optimum number of bins",
+    parameters = "values: RDD[NumericValue], numBuckets: Integer"
   )
-  def histogram[N: ClassTag](values: RDD[N], numBuckets: Int = 100)(implicit num: Numeric[N]): Unit = {
-    val (buckets, frequencies) = values.map(v => num.toDouble(v)).histogram(numBuckets)
-    histogram(buckets, frequencies)
+  def histogram[N: ClassTag](values: RDD[N], numBuckets: Integer = null.asInstanceOf[Integer])(implicit num: Numeric[N]): Unit = {
```

Comment:
I went for Java compatibility, where int won't be expected to be null. I see now that Int is also just another class, and not directly backed by java int, so I'll change it back.